packages/node-api/src/interfaces/default/types.ts

```diff
@@ -247,8 +247,7 @@ export interface StorageVersion extends Enum {
   readonly isV13AcknowledgeItems: boolean;
   readonly isV14HashLocPublicData: boolean;
   readonly isV15AddTokenIssuance: boolean;
-  readonly isV16MoveTokenIssuance: boolean;
-  readonly type: 'V1' | 'V2MakeLocVoid' | 'V3RequesterEnum' | 'V4ItemSubmitter' | 'V5Collection' | 'V6ItemUpload' | 'V7ItemToken' | 'V8AddSeal' | 'V9TermsAndConditions' | 'V10AddLocFileSize' | 'V11EnableEthereumSubmitter' | 'V12Sponsorship' | 'V13AcknowledgeItems' | 'V14HashLocPublicData' | 'V15AddTokenIssuance' | 'V16MoveTokenIssuance';
+  readonly type: 'V1' | 'V2MakeLocVoid' | 'V3RequesterEnum' | 'V4ItemSubmitter' | 'V5Collection' | 'V6ItemUpload' | 'V7ItemToken' | 'V8AddSeal' | 'V9TermsAndConditions' | 'V10AddLocFileSize' | 'V11EnableEthereumSubmitter' | 'V12Sponsorship' | 'V13AcknowledgeItems' | 'V14HashLocPublicData' | 'V15AddTokenIssuance';
```

Comment:
For some reason `V16MoveTokenIssuance` has vanished.

Reply:
This was actually because the `definitions.ts` file was not up to date (`types.ts` file has been fixed manually). I fixed that.